feed-universe.py

```diff
@@ -201,6 +206,10 @@ def slow_delete_all_topic_datasets(self, topic):
                         help='enable slow reset mode')
     parser.add_argument('-v', '--verbose', action='store_true', default=False,
                         help='enable verbose mode')
+    parser.add_argument('-t', '--topic-name', action='store_true', default='univers-ecospheres',
```

Comment:
Not convinced about requiring args vs conf, especially since we still have conf for the env. Is there a specific reason you went that way?

Reply:
No specific reason. Did not put it in conf because it's the same for both envs, but can do (or hardcode).

Reply:
I wouldn't hardcode, but you're right there isn't much variation, so both can work.
So far I tried to have "actions" as args and conf as conf, but the line sometimes is hard to draw...
:shrug: